src/sagemaker/mxnet/estimator.py

```diff
@@ -257,6 +252,8 @@ class constructor
         if not framework:
             # If we were unable to parse the framework name from the image it is not one of our
             # officially supported images, in this case just add the image to the init params.
+            init_params["framework_version"] = None
+            init_params["py_version"] = None
```

Comment:
I think it could be nice to attempt to parse these from the tag `if framework`

Reply:
sounds good